src/styles/blocks/header.scss

```diff
@@ -0,0 +1,22 @@
+.header {
+  box-shadow: 0 2px 4px $color-black-shadow;
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    max-width: 1200px;
```

Comment:
I think, you don't have to hardcore your width here

Reply:
it looks better on my 28" monitor))